lib/mini_racer/truffleruby.rb

```diff
@@ -0,0 +1,305 @@
+# frozen_string_literal: true
+
+require "monitor"
+
+module MiniRacer
+
+  class Context
```

Comment:
Could you rather integrate the code in the original file? Otherwise it's very difficult to review, understand the differences and it's also very brittle/easy to break

Reply:
Ah, those methods are all defined by the C extensions, they don't override something?

Reply:
Yes, it appears none of these methods override ruby code. They just implement the same as the C extension.